pyvista/core/datasetattributes.py

```diff
@@ -141,21 +388,67 @@ def t_coords(self, t_coords: np.ndarray):
         self.SetTCoords(vtkarr)
         self.Modified()
 
+    @property
+    def active_texture_name(self) -> Optional[str]:
```

Comment:
Shouldn't we also have an `active_texture` property?

Reply:
We should have a `active_t_coords` and then rename `active_texture_name` to `active_t_coords_name` in keeping with the VTK api.  We also have a DataSet level texture attribute that I don't want this confused with.

Reply:
Resolved in 22babd6894c947ff436f173bce9231d651813e66.